Azuka_Form.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Intro to HTML and Javascript</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+  <form>
+   <label>First name:</label><br>
+   <input type="text" id="first name" name="first name"
```

Comment:
your id and name attribute values can not have spaces. So instead of id="last name" try using id="last_name". There are a couple of attributes with this issue.